1-mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,9 +1,42 @@
-function setAlarm() {}
+function setAlarm() {
+  let inputVal = document.getElementById("alarmSet").value;
+  let titleEl = document.querySelector("#timeRemaining");
+  let time = splitTime(inputVal);
+  titleEl.innerText = `Time Remaining: ${formatVal(time.minute)}:${formatVal(
+    time.second
+  )}`;
+
+  let refreshIntervalId = setInterval(() => {
+    if (inputVal > 0) {
+      inputVal--;
+      time = splitTime(inputVal);
+      titleEl.innerText = `Time Remaining: ${formatVal(
+        time.minute
+      )}:${formatVal(time.second)}`;
+    }
+    if (inputVal === 0) {
+      inputVal = null;
+      playAlarm();
+      clearInterval(refreshIntervalId);
+    }
+  }, 1000);
+}
 
 // DO NOT EDIT BELOW HERE
 
 var audio = new Audio("alarmsound.mp3");
+function splitTime(inputVal) {
+  let minute = Math.floor(inputVal / 60);
+  let second = inputVal % 60;
+  return {
+    minute: minute,
+    second: second,
+  };
+}
```

Comment:
this is really good Doris - I like how you're breaking your code down into small reusable functions. JavaScript has a handy feature where instead of doing:
```
 return {
    minute: minute,
    second: second,
  };
```
if you your key and value are the same, you can just do this:
```
 return {
    minute,
    second
  };
```